core-it-support/maven-it-plugin-bootstrap/pom.xml

```diff
@@ -28,20 +28,25 @@ under the License.
     <version>2.1-SNAPSHOT</version>
   </parent>
 
-  <artifactId>core-it-bootstrap</artifactId>
+  <artifactId>maven-it-plugin-bootstrap</artifactId>
   <packaging>maven-plugin</packaging>
   <name>Bootstrap plugin</name>
 
+  <properties>
+    <maven.version>3.8.6</maven.version>
```

Comment:
Don't use this pattern. There is an official property `maven.version` which contains the version of the currently running Maven environment. Use, as we do now everywhere for versions camel case: `mavenVersion`.